flutter-studio/src/io/flutter/profiler/FlutterStage.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
```

Comment:
fix copyright notice here and elsewhere.

Reply:
Switched to Chromium copyright instead of Android Studio copyright.

Reply:
Done.